osf/models/node.py

```diff
@@ -344,17 +368,6 @@ def parent_node(self):
                 return parent
         return None
 
-    @property
-    def tag_names(self):
-        """
-        Optimization property. If node has been annotated with "annotated_tags"
-        in a queryset, use that value.  Otherwise, fetch the tags.
-        """
-        if hasattr(self, 'annotated_tags'):
-            return [] if self.annotated_tags == [None] else self.annotated_tags
-        else:
-            return self.tags.values_list('name', flat=True)
-
```

Comment:
Steve and I looked into this - this tag_names optimization was actually slowing things down after groups/guardian - we didn't know if it was an interaction with guardian, but this "optimization" has been removed.